### DISCUSSION TITLE
New Stability Clip.

### DISCUSSION THREAD

**Discussion by artificialguybr** [TheLastBen/fast-stable-diffusion#196] [Ideas]
From what I saw in the code, the Clip used is still the old one from Openai. Wouldn't it be interesting to upgrade to the new ones from Stability? For both Dreambooth (better training) and Automatic111?
Here u can see more:
https://laion.ai/blog/large-openclip/

The developer aircrumb already has a Colab on how to do it... You can use it as inspiration. https://github.com/aicrumb/doohickey

**Comment by grexzen** [comment]
LAION is not Stability, and the difference between L/14 and H/14 is minimal.
